### DISCUSSION TITLE
Minimum temperature parameter for junction?

### DISCUSSION THREAD

**Discussion by amayboudellal** [e2nIEE/pandapipes#428] [Q&A] [answered]
Hello,

I'm trying to run a flexibility analysis on a district heating grid where I would like to set a min and max temperature value for the junctions at which a sink is connected.

In pandapower, there are the min_vm_pu and max_vm_pu parameters that allow to set the min and max voltage at buses for which the power flow should solve and I was wondering if there was a similar feature in pandapipes.

Thanks for your help.

**Comment by jkisse** [comment, marked as answer]
Hi, 

the parameters you mentioned are only used for the optimal power flow in pandapower. In pandapipes, the is no optimal pipe flow yet. However, if you want to apply your own or an external solver, you can pass any value as a key word argument to the create-function and the respective columns will be automatically added to the `net.junction` table.

@SimonRubenDrauz knows about the latest planned developments and can help if you want to contribute to pandapipes.
